packages/components/src/base-control/stories/index.js

```diff
@@ -1,38 +1,81 @@
-/**
- * External dependencies
- */
-import { boolean, text } from '@storybook/addon-knobs';
-
 /**
  * Internal dependencies
  */
 import BaseControl from '../';
+import Button from '../../button';
+import { Spacer } from '../../spacer';
 import TextareaControl from '../../textarea-control';
 
 export default {
 	title: 'Components/BaseControl',
 	component: BaseControl,
-	parameters: {
-		knobs: { disable: false },
-	},
+	subcomponents: { BaseControl: BaseControl.VisualLabel },
```

Comment:
TIL about the `subcomponents` flag — it's probably something that we should use in all stories related to a component "family" — e.g. `Card`, `ItemGroup`, `ToolsPanel`...